app/metrics.yaml

```diff
@@ -1776,3 +1776,70 @@ app_theme:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+installation:
+  campaign:
+   type: string
+   send_in_pings:
+     - installation
+   description: >
+     Campaign
+   bugs:
+     - https://github.com/mozilla-mobile/fenix/issues/7295
+   data_reviews:
+     - https://github.com/mozilla-mobile/fenix/pull/8074
+   notification_emails:
+     - fenix-core@mozilla.com
+   expires: "2020-09-01"
+  network:
+    type: string
+    send_in_pings:
+      - installation
+    description: >
+      Network
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/7295
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8074
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  adgroup:
+    type: string
+    send_in_pings:
+      - installation
+    description: >
+      AdGroup
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/7295
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8074
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  creative:
+    send_in_pings:
+      - installation
+    type: string
+    description: >
+      Creative
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/7295
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8074
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  timestamp:
+    send_in_pings:
+      - installation
+    type: string
```

Comment:
This metric is called "timestamp", but it has a string type. What's the type of data going in there?

Reply:
The issue did not specify a date-time format so currently I'm just sending an unix timestamp obtained from the device as a string. Please let me know what format is required for the field.

Reply:
The Glean SDK has a series of [metric types](https://mozilla.github.io/glean/book/user/metrics/index.html) that can be used to represent data. Strings can be a last resort choice (given that they make no assumption on the underlying data) only if other types do not work :) So this really depends on the shape of this data coming in.

If all that's required is a date/time of when this ping was submitted, then you don't need to add anything, it's already being reported in the [ping_info section](https://mozilla.github.io/glean/book/user/pings/index.html) of glean pings.